.gitignore

```diff
@@ -43,3 +43,4 @@ integration/kubeconfig-*
 
 # Snaps
 eksctl_*_*.snap
+.DS_Store
```

Comment:
We try not to add environment specific files here. It's better that these are added to the `~/.gitignore_global` file instead.